### DISCUSSION TITLE
How to translate wind direction to degree number?

### DISCUSSION THREAD

**Discussion by STDataAndInformationService** [weather-gov/api#283] [Q&A]
The wind direction in the forecaster is [char], such as E, ENE, N, etc.

How can I convert it to degree number? such 345, 10, etc.

I found a post here https://www7.ncdc.noaa.gov/climvis/help_wind.html

Is this a correct reference?

thanks,

**Comment by KTibow** [comment]
Just Google it.

**Comment by scadergit** [comment]
Yikes...apparently noaa.gov used to be hosted by geocities. 

Lots of options online. Some existing libraries that do it for you, but not sure which language you're using. Most simply find the index of an array of compass names then multiply the index by 22.5 (which is 360/16).
